### ISSUE TITLE
Meet America's Two Grossest Hotels

### ISSUE THREAD

**Issue by angelesyw764** [angelesyw764/angeles#12]
<p><strong>Meet America's Two Grossest <b>Hotels</b></strong><br>
Old and getting older, the Hotel Carter looks blank rather than dramatic, boring and sad rather than creepy or shifting or absurd. But there, seducing tourists in search of a cheap rate in the middle of the street-level scrum, is the one glittering &hellip;<br><i>Read more on <a target="_blank" rel="nofollow" href="http://ift.tt/1BbXKpS">Slate Magazine</a><br><br></i></p>
<p><strong>Google's Mobile <b>Search</b> Strategy: Bake In and Take Out</strong><br>
The first piece was a pair of new ad formats for cars and hotels, bringing flashier photo spreads and easier booking within search, expanding on its entrance into hospitality booking last month. With those, Google also teased the launch of a new &hellip;<br><i>Read more on <a target="_blank" rel="nofollow" href="http://ift.tt/1zOnk8Q">Re/code</a><br><br></i></p>
<p><strong>11 Cool Travel Businesses You Need to <b>Check</b> Out</strong><br>
Finding an inexpensive flight, booking the perfect hotel room, planning a fun itinerary, funding your trip &mdash; these are all things that make going on vacation more difficult than you'd like it to be. The good news is, there are plenty of businesses out &hellip;<br><i>Read more on <a target="_blank" rel="nofollow" href="http://ift.tt/1Px74QT">Business News Daily</a><br><br></i></p>
<p><strong>Easthampton fire chief accuses Cottage Square contractors of lying to get <b>&hellip;</b></strong><br>
By Tuesday, the first wave of tenants, some of whom had been living in hotels since April 1, were pulling cars into the parking lot, meeting with an on-site manager to sign lease documents, and moving belongings into their new apartments. Henneman said &hellip;<br><i>Read more on <a target="_blank" rel="nofollow" href="http://ift.tt/1F1yHaB">MassLive.com</a><br><br></i></p>
<p><strong>Morgans <b>Hotel</b> Group Announces Leadership Changes</strong><br>
The Board intends to continue its CEO search and in the interim, Richard Szymanski, Chief Financial Officer, will temporarily assume the duties of the principal executive officer for securities law reporting purposes. "On behalf of the entire Board, I &hellip;<br><i>Read more on <a target="_blank" rel="nofollow" href="http://ift.tt/1AdNQJv">PR Newswire (press release)</a><br><br></i></p>
<p><strong>A New Marriott App Connects Guests With <b>Hotels</b></strong><br>
Mobile Request is intended to save guests time and make their stay more efficient, according to Matthew Carroll, vice president of Marriott Hotels. &ldquo;You don't have to search for the hotel's number or wait on hold for a staff member to be free,&rdquo; he said.<br><i>Read more on <a target="_blank" rel="nofollow" href="http://ift.tt/1Iy1NEC">New York Times</a><br><br></i></p>
<p>The post <a rel="nofollow" href="http://ift.tt/1Px76s4">Meet America's Two Grossest <b>Hotels</b></a> appeared first on <a rel="nofollow" href="http://ift.tt/1sHHUEy">Discounted Hotel Rooms</a>.</p>
<br><br>
via WordPress http://ift.tt/1Px76s4
